tests/components/knx/test_expose.py

```diff
@@ -0,0 +1,67 @@
+"""Test knx expose."""
+from unittest.mock import AsyncMock, Mock
+
+from homeassistant.components.knx.expose import KNXExposeSensor
+
+
+async def test_binary_expose(hass):
+    """Test that a binary expose sends only telegrams on state change."""
+    e_id = "fake.entity"
+    xknxMock = Mock()
+    xknxMock.telegrams = AsyncMock()
+    KNXExposeSensor(hass, xknxMock, "binary", e_id, None, "0", "1/1/8")
```

Comment:
We should not interact with integration details in the tests. Set up the integration and then assert some core state.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations